packages/components/molecules/f-mega-modal/README.md

```diff
@@ -119,10 +76,6 @@ The props that can be defined are as follows:
 
 ### CSS Classes
 
-The modal has its own styles which are scoped to the component using CSS modules to prevent conflicts with existing styles on the page.
```

Comment:
Same here as above – I'd keep this description in.

Reply:
This is true for all Vue components, there are scoped and public properties.  The reader doesn't need to be reminded of it on every readme.  It could go in the root documentation if it should go anywhere, though this document isn't an instruction of how to use Vue it's just about how to use this specific document.

Reply:
This won't be in every component. Not all components have stlyes that are exposed in this way as in the modal.

Reply:
We don't document the internal styles though.  A sentence effectively saything that there's styles that you're not able to use is more likely to be confusing than helpful.